static/js/containers/LoginButton.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import DropdownButton from 'react-bootstrap/lib/DropdownButton';
+import { LinkContainer } from 'react-router-bootstrap';
```

Comment:
Can you use this syntax instead?

`import LinkContainer from 'react-router-bootstrap/lib/LinkContainer';`

Importing modules directly is best in case there's other stuff in `react-router-bootstrap` which webpack can leave out